libs/platform/src/lib/components/approval-flow/approval-flow.component.ts

```diff
@@ -352,13 +379,9 @@ export class ApprovalFlowComponent implements OnInit, OnDestroy {
 
     /** @hidden Fetch all necessary data and enter edit mode */
     _enterEditMode(): void {
-        this._editModeInitSub = forkJoin([
-            this.dataSource.fetchWatchers().pipe(take(1)),
-            this.dataSource.fetchTeams().pipe(take(1))
-        ])
-            .subscribe(([users, teams]) => {
-                this._usersForWatchersList = users;
-                this._teams = teams;
+        this._editModeInitSub = this.dataSource.fetchWatchers().pipe(take(1))
```

Comment:
The same as in my first comment (you do not need `_editModeInitSub` since you have `take(1)`)

Reply:
First value may be received after the component has destroyed. In that case there would be a memory leak.